assets/src/util/streetViewUrl.ts

```diff
@@ -0,0 +1,12 @@
+export const streetViewUrl = ({
+  latitude,
+  longitude,
+  bearing,
+}: {
+  latitude: number
+  longitude: number
+  bearing?: number
+}): string =>
+  `https://www.google.com/maps/@?api=1&map_action=pano&viewpoint=${latitude}%2C${longitude}&heading=${
+    bearing || 0
```

Comment:
**suggestion (non-blocking):** it might make sense to leave this blank if no `bearing` is specified rather than defaulting to 0, since the [documentation](https://developers.google.com/maps/documentation/urls/get-started#street-view-action) indicates that it's optional.